### PR TITLE
Bar Drink: Monster Mash

### PR DESCRIPTION
**Pull request by GreatBigFailure**
A new bar drink: Monster Mash. Made with 1u ectoplasm, 1u caramel, and 1u tomato juice.
Sprites by Blithering, it's a very cute pumpkin. 

mosntermash

If you aren't in the halloween spirit it won't have any effect byond getting you graveyard smashed. If you are in the spooky spirit it'll have an effect based on your costume. In fact all 3 effects can be active at once if you're really eager to trick or treat!

This PR also adds a bus only tank of the stuff, 1000u.
graveyardsmash

-->
:cl:
 * rscadd: The Monster Mash drink can now be made at the bar, how terrifying!

### CODE REVIEW

**Comment by SonixApache** [comment]
both sprites are adorable

**Comment by GreatBigFailure** [comment]
The bat flaps its wings and it's adorable. 
Oh I should also probably mention that this finally uses the scarybat sprite we've had for ages, the fat cute bat. 
creditbat
This guy. So uh, credit to whoever made that however long ago, great work mystery man.

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -7516,6 +7516,51 @@
...
+	playsound(H, 'sound/effects/rattling_bones.ogg', 100, 1)
+	if(prob(15))
+		H.emote("spin")
+		H.visible_message("<span class='good'>[H] does the mash!</span>")
+		if(prob(25))
+			spawn(1 SECONDS)
+				H.emote("spin")
+				H.visible_message("<span class='good'>[H] does the monster mash!</span>")
+
+/datum/reagent/ethanol/drink/monstermash/proc/joltOfMyElectrodes(mob/living/carbon/human/H)
```
Comment (ShiftyRail) [inline code/modules/reagents/Chemistry-Reagents.dm:7551]: Very powerful plasma fires vibes here (jk)

File: code/modules/mob/living/simple_animal/hostile/human/halloween.dm [hunk 2]
```diff
@@ -645,3 +645,16 @@
...
+
+
+/mob/living/simple_animal/dracson
+	name = "Young Vampire Bat"
+	desc = "Whatever happened to my Transylvania Twist?"
+	icon = 'icons/mob/animal.dmi'
+	icon_state = "scarybat-dance"
+	mob_property_flags = MOB_UNDEAD | MOB_SUPERNATURAL
+
+/mob/living/simple_animal/death(var/gibbed = FALSE)
```
Comment (Exxion) [inline code/modules/mob/living/simple_animal/hostile/human/halloween.dm]: Somehow I don't think this is the right path

Comment (GreatBigFailure) [reply code/modules/mob/living/simple_animal/hostile/human/halloween.dm]: Oh lord
I moved that guy around so many times deciding where to put him I guess I lost a chunk along the way.

**Pull request merged**